server/app/services/auth_provider.rb

```diff
@@ -89,18 +88,46 @@ def update_kontena
         }
       }
     }
-
     response = client.request(
       :put,
       uri.to_s,
       header: {
         'Content-Type' => 'application/json',
-        'Accept' => 'application/json'
+        'Accept' => 'application/json',
+        'Authorization' => "Bearer #{master_access_token}"
       },
       body: body.to_json
     )
   end
 
+  def master_access_token
```

Comment:
this should probably be in config, something like @master_access_token ||= config['cloud.master_token_secret'] ||= get_master_access_token 

(anything ending with _secret is stored encrypted in the db)